index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact">
+        <h3 id="form-header"> I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+        <form class="form" action=" https://formspree.io/f/xyylppbj" method="POST">
+          <input type="text" name="Names"  id="name" maxlength='30' placeholder="Name">
+          <input type="email" name="E-mail" id="email" placeholder="Email">
+          <textarea maxlength='500' name="Message" rows="5" id="message" placeholder="Your message"></textarea>
+          <input type="submit" value="Get In Touch">
+        </form>
+      </section>
```

Comment:
[ Optional ] Please follow the Figma design and add the social media navigation icon as  links to those social median websites